No javascript message for time travel

alex-page opened the pull request:
Currently when a user doesn't have javascript enabled when opening the time travel section the site looks broken.

This pull request removes the history section if there is no javascript. It also adds a simple message with a button to return to the site.

Commit:
No javascript message for timetravel

prichodko commented:
Definitely nice thing to have! But there is still some room for improvement.

Issues:
- use same font as on the initial page
- fix capitalization of GitHub and JavaScript names

Ideas:
- center vertically?
- increase size of emojis?

What do you think?

alex-page commented:
Sounds good to me, I will make those adjustments :)

Commit:
Merge branch 'master' into nojs-timetravel

Commit:
Restructuring some base styles

Commit:
Adding variables to timetravel

Commit:
Add microsoft logo, make layout similar to home

alex-page commented:
Hey @prichodko thanks for the feedback, they are all great suggestions.

Here is what I have worked on:
- My previous pull request #125  has fixed up the font families not being the same.
- Fixed up capitilisation
- Instead of making the emojis bigger I gave the no-js section a useful title
- I added microsoft logo in the no-js section
- I added the variables and replaced a bunch of inconsistent styles between time-travel and homepage

I decided not to do vertical centering. After implementing it with flex box I felt the page would work better trying to match the user experience  on the home page. To do this I pulled across the variables and reworked some of the CSS in the styles and time-travel to be more consistent.

Commit:
Fix incorrect formatting
